auto_install/install.sh

```diff
@@ -2429,10 +2573,14 @@ if \$programname == 'ovpn-server' then stop" | $SUDO tee /etc/rsyslog.d/30-openv
 }" | $SUDO tee /etc/logrotate.d/openvpn > /dev/null
 
 	# Restart the logging service
-	case ${PLAT} in
-		Debian|Raspbian|Ubuntu)
+	case "${PLAT}" in
+		Debian | Raspbian | Ubuntu)
 			$SUDO systemctl -q is-active rsyslog.service && $SUDO systemctl restart rsyslog.service
 		;;
+		Alpine)
+			"${SUDO}" rc-service -is rsyslog restart
+			"${SUDO}" rc-service -iN rsyslog start
```

Comment:
If sudo is not installed, "${SUDO}" will be empty, so the script will try to run `"" rc-service -is rsyslog restart`. Like before, it should be unquoted (there are many instances).